### ISSUE TITLE
cant call pie

### ISSUE THREAD

**Issue by bayu2208** [dumbeau/AutoHotPie#163]
i use logitech mx master 3s and logitech option plus to customized shortcut, i cant call autohotipie with that

**Comment by Da3n0n** [comment]
yeah mate for me i have now bought 3 Logitech mouse , i used to have g305 worked perfect in auto hotpie , i switched to MX master thinking productivity gonna be next level , NO so wrong , yeah the mx jsut sits there now becasue it useless to me ,  this isn't an issue with autohotpie its the way key down and key up events are triggered on the MX series  , it detects events like a click instead of hold , but all g series moue buttons do detect bot down and up event and held . i bought g502 x lightspeed and honestly its the best decision i have made when it comes to perihperals,  the 5 extra keys  way more valuable than side scroll espcially if you plan on using pie menus :P i hope this help.

**Comment by dumbeau** [comment]
So the issue is that the MX Master sends key_down and key_up immediately when you press down the button, as opposed to sending key_up when you release the button.

@bayu2208 I was told about this potential fix: https://www.reddit.com/r/logitech/comments/103ovk2/mx_master_3_mapped_buttons_wont_hold_after/

If you could give this a shot, lmk if this fixes anything.
